### PR TITLE
EZP-29838: Cannot use Multi-file upload on Microsoft Edge

### PR DESCRIPTION
**Pull request by tischsoic**
JIRA: https://jira.ez.no/browse/EZP-29838

Fixes multi-file upload not working on MS Edge.

The error is caused by spread operator, because `list.files` is not an array but rather `FileList`. Problem can be solved by using `Array.from(list.files)` instead of `[...list.files]`.

### CODE REVIEW

**Review by dew326 — APPROVED** [review]

File: src/modules/multi-file-upload/multi.file.upload.module.js [hunk 1]
```diff
@@ -143,14 +143,14 @@ export default class MultiFileUploadModule extends Component {
...
+     * @param {Event} event
      * @returns {Array}
      * @memberof MultiFileUploadModule
      */
-    proccessUploadedFiles(event) {
+    processUploadedFiles(event) {
         const list = this.extractDroppedFilesList(event);
 
-        return [...list.files].map((file) => ({
+        return Array.from(list.files).map((file) => ({
```
Comment (sunpietro) [inline src/modules/multi-file-upload/multi.file.upload.module.js:153]: Wouldn't it be better to use `[].slice.call(list.files).map()` instead?

**Review by sunpietro — APPROVED** [review]

File: src/modules/multi-file-upload/multi.file.upload.module.js (hunk 1 above)
Comment (tischsoic) [reply src/modules/multi-file-upload/multi.file.upload.module.js:153]: Are you thinking about browser compatibility or something else?
I checked MDN and `Array.from` seems to be supported on all modern browsers (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/from#Browser_compatibility) so in my opinion it's OK to use it here. 
What do you think?

Comment (sunpietro) [reply src/modules/multi-file-upload/multi.file.upload.module.js:153]: I'm thinking about perfomance

Comment (tischsoic) [reply src/modules/multi-file-upload/multi.file.upload.module.js:153]: I don't think that we need to optimise this one line. 😄 
And, for me, this notation: `Array.from(list.files)` is better than this: `[].slice.call(list.files)` when it comes to code readability. :)

Comment (sunpietro) [reply src/modules/multi-file-upload/multi.file.upload.module.js:153]: I'm fine with that

**Review by micszo — APPROVED** [review]

**Comment by lserwatka** [comment]
@tischsoic please merge it up

**Comment by tischsoic** [comment]
Merged up to master: https://github.com/ezsystems/ezplatform-admin-ui-modules/commit/b5fa167d7de6c63c3f969e1a23ad41177806d92e

**Pull request merged**